BlocQuery/BlocQuery/ViewController.m

```diff
@@ -8,19 +8,18 @@
 
 #import "ViewController.h"
 
-@interface ViewController ()
+@interface ViewController () <PFLogInViewControllerDelegate>
 
 @end
 
 @implementation ViewController
 
-- (void)viewDidLoad {
-    [super viewDidLoad];
-
-    PFObject *testObject = [PFObject objectWithClassName:@"TestObject"];
-    testObject[@"foo"] = @"bar";
-    [testObject saveInBackground];
-
+- (void)viewDidAppear:(BOOL)animated {
+    [super viewDidAppear:YES];
+    
+    PFLogInViewController *logInVC = [[PFLogInViewController alloc] init];
```

Comment:
You want to do a few more things here:
1. Check if the user is already logged in. In that case you don't need to present the login view
2. Once the user logged in you want to send them somewhere
3. You didn't actually implement the delegate.